### ISSUE TITLE
Unable to download baseline checkpoint files

### ISSUE THREAD

**Issue by csherstan** [google/dopamine#48]
I've been trying to download the various checkpoint files as per the instructions at the bottom of: [https://github.com/google/dopamine/tree/master/docs](https://github.com/google/dopamine/tree/master/docs). 

I've tried numerous variations of agents, games, runs and suffixes but every time I receive an error stating there is no object by that name.

I am typing the urls directly into Chrome.

Here's an example that I've tried:
`https://storage.cloud.google.com/download-dopamine-rl/lucid/dqn/qbert/1/tf_ckpt-199.index`

I assumed you used the same naming for the games as the names given in the dropdown on [https://google.github.io/dopamine/baselines/plots.html](https://google.github.io/dopamine/baselines/plots.html).

**Comment by psc-g** [comment]
hi,
sorry for the confusion.
the naming in the dropdown is all small-cased. you need to use camel-cased names.
for the example you provide it would be:
https://storage.cloud.google.com/download-dopamine-rl/lucid/dqn/Qbert/1/tf_ckpt-199.index

can you verify whether this works?

**Comment by csherstan** [comment]
Yes, that works. Can you update the documentation for this? Thanks.

**Comment by csherstan** [comment]
Alternatively, perhaps the downloads should all be made lower case so no one has to guess at the camel-case. I would have gone with QBert :)

**Comment by yakotaki** [comment]
For GCP
!gsutil -q -m cp -R gs://download-dopamine-rl/lucid/dqn/Qbert/1/tf_ckpt-199.index /where_to_copy/

**Comment by mignonjia** [comment]
> hi, sorry for the confusion. the naming in the dropdown is all small-cased. you need to use camel-cased names. for the example you provide it would be: https://storage.cloud.google.com/download-dopamine-rl/lucid/dqn/Qbert/1/tf_ckpt-199.index
> 
> can you verify whether this works?

Hi, this line of download link doesn't work now (as of date 05/15/2026), which shows "Forbidden Error 403". 
Also, the https://storage.cloud.google.com/download-dopamine-rl/dqn_checkpoints.tar.gz links doesn't work either.

Could you please check this issue? Thx!
